docs/DeveloperGuide.md

```diff
@@ -81,16 +91,48 @@ CLIrcuit Assistant solves three problems:
 |v1.0|new CEG/EE Student | change values assigned to the components| obtain different calculations quickly|
 |v1.0|new CEG/EE Student|add components arranged in either series or parallel|obtain my configuration for analysis|
 |v1.0|new CEG/EE Student |calculate/verify my calculations of effective resistance, total capacitance, etc.|quickly and accurately do so via CLI, and not 3rd party software that I have to install/load online|
-|v2.0|CEG/EE Student|compute a boolean logical expression|familiarise myself with logical computation used in digital circuits|
+|**v2.0** (latest)|CEG/EE Student|compute a boolean logical expression|familiarise myself with logical computation used in digital circuits|
 
-## Non-Functional Requirements
+### Non-Functional Requirements
 
-{Give non-functional requirements}
+1. Should work on any *mainstream OS* as long as it has Java 11 or above installed.
+2. The average user profile is assumed to be someone who possesses *at least a basic understanding of the fundamental concepts of digital circuits, its components, and boolean logic computation* (concepts covered in CG1111, CS1231 and EE2026).
+3. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
 
 ## Glossary
 
 * *glossary item* - Definition
 
-## Instructions for manual testing <a name="instructions"></a>
+## Appendix: Instructions for manual testing
+### Initial launch  
+1. Download the jar file and copy it into an empty folder
+2. Open a new terminal window and navigate to the same directory where your ip.jar is located
+3. Enter the command `java -jar duke.jar` into the terminal window to launch the application
+4. Enter `help` to go into interactive tutorial mode or `summary` to generate a summary of all the commands
+
+### Creating a digital circuit  
+1. Enter `template` followed by the template type, e.g.: `template rc`
+There are four templates to choose from: `r`,`rc`,`rl`,`lc`. 
+2. Enter `set` followed by the component type to set the component value, e.g.: `set r 500`
+The four types of components are: `r`,`c`,`l`,`v`. Note that the value inputted can be an integer or float
+3. Enter `add` followed by the configuration, component , and component value to add a component e.g.:`add parallel c 500`
+
+### Creating a logic gate 
+Similar to the creation of a digital circuit, we create a `template`, `set` values, and can `add` values.
+
+For the detailed steps, visit: https://ay2021s1-cs2113t-w13-3.github.io/tp/UserGuide.html#logic-gate-commands
+
+### Calculating values
+For calculation of values, the `calc` command is used.
+
+Detailed steps on calculation for 
```

Comment:
Consider using bullet points here? Possibly standardise the numbering/bulleting in this Appendix section?

Reply:
sounds good